modules/cco-ccoctl-creating-at-once.adoc

```diff
@@ -175,6 +175,7 @@ ifdef::aws-sts,google-cloud-platform,azure-workload-id[]
 +
 endif::aws-sts,google-cloud-platform,azure-workload-id[]
 ifdef::aws-sts[]
+
```

Comment:
This stray line has broken the formatting, need to remove
```suggestion
```
![image](https://github.com/user-attachments/assets/ff44870b-09d7-4fd4-81f9-2524c5803ac1)